[2021-08-22 15:36:56,964] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: download_rocket_launches.get_pictures 2021-08-20T00:00:00+00:00 [queued]>
[2021-08-22 15:36:57,289] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: download_rocket_launches.get_pictures 2021-08-20T00:00:00+00:00 [queued]>
[2021-08-22 15:36:57,289] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-22 15:36:57,289] {taskinstance.py:1088} INFO - Starting attempt 2 of 2
[2021-08-22 15:36:57,290] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-22 15:36:57,378] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): get_pictures> on 2021-08-20T00:00:00+00:00
[2021-08-22 15:36:57,390] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'download_rocket_launches', 'get_pictures', '2021-08-20T00:00:00+00:00', '--job-id', '115', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/1.rocket_launch_data.py', '--cfg-path', '/tmp/tmpvxf44mr9', '--error-file', '/tmp/tmp8sqdqwbk']
[2021-08-22 15:36:57,384] {standard_task_runner.py:52} INFO - Started process 3202 to run task
[2021-08-22 15:36:57,391] {standard_task_runner.py:77} INFO - Job 115: Subtask get_pictures
[2021-08-22 15:36:57,472] {logging_mixin.py:104} INFO - Running <TaskInstance: download_rocket_launches.get_pictures 2021-08-20T00:00:00+00:00 [running]> on host b4dfeeeb641c
[2021-08-22 15:36:57,578] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=download_rocket_launches
AIRFLOW_CTX_TASK_ID=get_pictures
AIRFLOW_CTX_EXECUTION_DATE=2021-08-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-20T00:00:00+00:00
[2021-08-22 15:36:57,580] {python.py:151} INFO - Done. Returned value was: None
[2021-08-22 15:36:57,610] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=download_rocket_launches, task_id=get_pictures, execution_date=20210820T000000, start_date=20210822T153656, end_date=20210822T153657
[2021-08-22 15:36:57,740] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-22 15:36:57,768] {local_task_job.py:149} INFO - Task exited with return code 0
